ndk/src/user/index.ts

```diff
@@ -1,16 +1,16 @@
 import { nip05, nip19 } from "nostr-tools";
 import { NDKEvent, NDKTag, NostrEvent } from "../events/index.js";
-import {NDK} from "../ndk/index.js";
+import { NDKRelayList } from "../events/kinds/NDKRelayList.js";
```

Comment:
This is all formatting changes. I'm not sure how we can get our prettier/vscode setups working the same... this shit is such a pain.